test_runner/src/test/kotlin/ftl/json/SavedMatrixTest.kt

```diff
@@ -197,6 +197,7 @@ class SavedMatrixTest {
         assertEquals(expectedOutcome, savedMatrix.outcome)
         assertEquals(expectedOutcomeDetails, savedMatrix.testAxises.first().details)
         assertEquals(INVALID, savedMatrix.state)
+        assertEquals("Matrix: [${testMatrix.testMatrixId}] failed: Unknown error", savedMatrix.errorMessage())
```

Comment:
We could have error message hardcoded (or as a variable) then `savedMatrix.errorMEssage` could be private

Reply:
Could you explain?

Reply:
Sure! (actually, it's like the second part of https://github.com/Flank/flank/pull/1159/files#r495712155 comment)

We know what is matrix outcome so instead of calling `savedMatrix.errorMessage()` we could have this message hardcoded in assertion or have it extracted into variable `val expectedErrorMessage = ...`

Reply:
@adamfilipow92 skip what I wrote above

Reply:
Hmm but making ```errorMessage()``` private and test validation in ```MatrixMap``` instead on ```errorMessage()```  is a good idea. WDYT?